src/po/sr.po

```diff
@@ -4971,6 +4880,80 @@ msgstr " ИЗБОР БЛОКА"
 msgid "recording"
 msgstr "снимање"
 
+#, c-format
+msgid "Searching for \"%s\" in \"%s\""
+msgstr "Тражи се \"%s\" у \"%s\""
+
+#, c-format
+msgid "Searching for \"%s\""
+msgstr "Тражи се\"%s\""
+
+#, c-format
+msgid "not found in '%s': \"%s\""
+msgstr "није пронађено у '%s': \"%s\""
+
+msgid "Source Vim script"
+msgstr "Изворна Vim скрипта"
```

Comment:
Is "source" here an adjective or a verb?

Reply:
You're right, this appears to be a verb.
I'm having difficulty to find an appropriate translation of "source" in this context.
The same applies for "fold, folding". Podvijanje, svijutak sounds weird… suggestions are welcome.

Reply:
Yeah, in casual conversation I'm used to using "sorsovati" which is most definitely not a verb in Serbian. The problem is that the verb "source", at least as far as I'm aware, is highly technical and not applicable outside of scripting languages. I don't think Serbian has an appropriate word for it. The closest I can think of is "ucitati" which is "load" and is wrong.

"Folding" == "preklapanje"? Then "to fold" would be "preklopiti" and "a fold"... "preklop"? I don't think that last one is a word.